service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/mapping/JsonResponseMapper.java

```diff
@@ -320,18 +332,36 @@ private Object convertType(ResponseWriteContext writeContext, Object item) {
     return typeConverterRouter.convert(item, writeContext.getParameters());
   }
 
+  private void validateObjectData(ResponseWriteContext context, String path, Map<String, Object> data) {
```

Comment:
```suggestion
  private void validateRequiredPropertiesMapping(ResponseWriteContext context, String path, Map<String, Object> data) {
```
Beschrijft imho iets beter wat de methode functioneel doet

Reply:
done